README.md

```diff
@@ -60,28 +60,40 @@ The extension allows developers to find and fix security vulnerabilities in thei
 about the status of their code by continuously scanning it locally with [JFrog Xray](https://jfrog.com/xray/).
 
 ### What security capabilities do we provide?
-#### Software Composition Analysis (SCA)
-Scan your project dependencies for security issues.
-For selected security issues, get leverage-enhanced CVE data that is provided by our JFrog Security Research team.
-To learn more about enriched CVEs, see [here](https://www.jfrog.com/confluence/display/JFROG/JFrog+Security+CVE+Research+and+Enrichment)
+####  Basic
+##### 📦 Software Composition Analysis (SCA)
+- Scan project dependencies for security issues
+- Access enriched CVE data provided by the JFrog Security Research team.
 
-#### Advanced Scans
+Learn more about enriched CVEs in the [JFrog Security CVE Research and Enrichment](https://www.jfrog.com/confluence/display/JFROG/JFrog+Security+CVE+Research+and+Enrichment) documentation.
+
+####  Advanced
 *Requires Xray version 3.66.5 or above and Enterprise X / Enterprise+ subscription with Advanced DevSecOps.*
 
-With advanced [**Contextual Analysis**](#contextual-analysis), understand the applicability of CVEs in your application and utilize JFrog Security scanners to analyze the way you use 3rd party packages in your projects.
+##### 🔍 Contextual Analysis
+[Contextual Analysis](#viewing-vulnerability-details), understand the applicability of CVEs in your application and utilize JFrog Security scanners to analyze the way you use 3rd party packages in your projects.
 Automatically validate some high-impact vulnerabilities, such as vulnerabilities that have prerequisites for exploitations, and reduce false positives and vulnerability noise with smart CVE analysis.
 
 To learn more, see [here](https://www.jfrog.com/confluence/display/JFROG/Vulnerability+Contextual+Analysis).
```

Comment:
To learn more, see [here](https://www.jfrog.com/confluence/display/JFROG/Vulnerability+Contextual+Analysis).
-->
Learn more about [Vulnerability Contextual Analysis](https://www.jfrog.com/confluence/display/JFROG/Vulnerability+Contextual+Analysis).